internal/deviations/deviations.go

```diff
@@ -70,6 +70,8 @@ var (
 	InterfaceEnabled = flag.Bool("deviation_interface_enabled", false,
 		"Device requires interface enabled leaf booleans to be explicitly set to true.  Full OpenConfig compliant devices should pass both with and without this deviation.")
 
+	IPV4Enabled = flag.Bool("deviation_ipv4_enabled", false, "Device does not support interface/ipv4/enabled leaf. The ipv4 is enabled when it is configured.")
```

Comment:
Deviation should use false value to indicate compliant behavior, and true value when device is not compliant. See the package doc above.

Therefore, we should do something like this:

```suggestion
	IPv4MissingEnabled = flag.Bool("deviation_ipv4_missing_enabled", false, "Device does not support interface/ipv4/enabled, so suppress configuring this leaf.")
```

Reply:
thank you! addressed